packages/orbit-components/src/InputSelect/index.tsx

```diff
@@ -436,8 +443,8 @@ const InputSelect = React.forwardRef<HTMLInputElement, Props>(
               label={label}
               help={help}
               error={error}
-              onFocus={() => setIsOpened(true)}
-              readOnly
+              onFocus={!readOnly ? () => setIsOpened(true) : undefined}
```

Comment:
This is correct, but we still need to call `onFocus` if it exists.
Basically, by calling `setIsOpened` it would open the modal and the modal would autofocus the inner input, calling `onFocus` if it exists. But in this scenario we don't open the modal. However, we still need to call the `onFocus` if it's defined.